index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <!--Mystery favicon-->
+  <link href="./assets/favicon.ico" rel="icon" type="image/x-icon" />
+  <!--Google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Arvo|Lato" rel="stylesheet">
+  <link rel="stylesheet" href="style.css" />
+  <title>Mystery cluedo project</title>
+</head>
+
+<body>
+  <h1>Welcome to the myster cluedo game! </h1>
+  <h2>Choose a killer, a weapon and a room and then reveal the crime </h2>
+  <section class="cards">
+    <article class="killer-deck" id="killerCard">
+      <h2>KILLER</h2>
+      <p id="killerName"></p>
+      <p id="killerAge"></p>
+      <p id="killerOccupation"></p>
+      <p id="killerDescription"></p>
+      <div class="killerImg-container" id="killerImgContainer">
+        <img id="killerImage" src="./assets/icons8-inquiry-100.png" height="100" src="" alt="Killer" />
+      </div>
+
+    </article>
+    <article class="weapon-deck" id="weaponCard">
+      <h2>WEAPON</h2>
+      <p id="weaponName"></p>
+      <p id="weaponWeight"></p>
+      <img id="weaponImage" src="./assets/icons8-weapons-100.png" height="100" src="" alt="Weapon" />
+    </article>
+    <article class="room-deck" id="roomCard">
+      <h2>ROOM</h2>
+      <p id="roomName"></p>
+      <img id="weaponImage" src="./assets/icons8-prefab-house-100.png" height="100" src="" alt="House" />
+    </article>
+  </section>
+
+  <h2 id="mystery"></h2>
```

Comment:
Love the style, just a little hard to read the mystery text here, maybe add a background or change the font color. I always struggle with this when having text over image ;)

Reply:
I agree :) Changed the font color to red.